openedx/core/djangoapps/programs/utils.py

```diff
@@ -351,4 +351,12 @@ def supplement_program_data(program_data, user):
             # TODO: Currently unavailable on LMS.
             run_mode['marketing_url'] = ''
 
+            certificate_data = certificate_api.certificate_downloadable_status(user, course_key)
+            certificate_uuid = certificate_data.get('uuid')
+            if certificate_uuid:
```

Comment:
@mattdrayer as did I, which is why I tagged you for review 😎 . Thanks for the feedback!